pages.ne/android/bugreportz.md

```diff
@@ -1,4 +1,4 @@
-# Bugreport
+# Bugreportz
```

Comment:
```suggestion
# bugreportz
```

The title doesn't match with the English page (if the commands aren't capitalized and don't support capitalization, we shouldn't capitalize it IMO)